src/js/components/Button/StyledButton.js

```diff
@@ -144,11 +166,7 @@ const StyledButton = styled.button`
 
   ${props => !props.disabled && props.active && activeStyle}
   ${props =>
-    props.disabled &&
-    props.theme.button &&
-    disabledStyle(
-      props.theme.button.disabled && props.theme.button.disabled.opacity,
-    )}
+    props.disabled && props.theme.button.disabled && disabledStyle(props)}
```

Comment:
```suggestion
    props.disabled &&  props.theme.button && props.theme.button.disabled && disabledStyle(props)}
```

Reply:
I see the check was added, but why did we remove the other check for `disabled`?

Reply:
Whoops, let me add that back in.